extra/ldr-to-hdr.sh

```diff
@@ -0,0 +1,226 @@
+	exit 1
+fi
+if [[ -z $CONFIG_VIG_PATH ]]; then
+	echo "ERROR: No vignetting config file given, please give file."
+	exit 1
+fi
+if [[ -z $CONFIG_FISHEYE_PATH ]]; then
+	echo "ERROR: No fisheye correction config file given, please give file."
+	exit 1
+fi
+if [[ -z $CONFIG_CAMRESP_PATH ]]; then
+	echo "ERROR: No camera response config file given, please give file."
+	exit 1
+fi
+if [[ -z $CONFIG_FISHEYE_DIAM || -z $CONFIG_CROP_X || -z $CONFIG_CROP_Y || -z $CONFIG_VA_V || -z $CONFIG_VA_H || -z $CONFIG_OUT_X || -z $CONFIG_OUT_Y ]]; then
+	echo "ERROR: Camera settings file did not give all information needed! Please check it."
+	exit 1
+fi
+
+# Set up the temp files, something simple.
+TEMP_FILENAME="$TEMP_FILEPATH/$(md5sum ${INPUT_FILES[0]} | awk '{ print $1 }').hdr"
+TEMP_FILENAME_ALT="$TEMP_FILEPATH/$(md5sum ${INPUT_FILES[1]} | awk '{ print $1 }').hdr"
+
+# Make sure neither file already exists
+rm $TEMP_FILENAME 2> /dev/null
+rm $TEMP_FILENAME_ALT 2> /dev/null
+
+# One last check, let's make sure that the Radiance suite and hdrgen are in PATH.
+
+# hdrgen
+if ! $(which hdrgen 2>&1 > /dev/null); then
+	echo "ERROR: Missing PATH to hdrgen, please add to your PATH or set with -h flag."
+	exit 1
+fi;
+
+# pcomb from Radiance
+if ! $(which pcomb 2>&1 > /dev/null); then
+	echo "ERROR: Missing PATH to Radiance utilities, please add to your PATH or set with -r flag."
+	exit 1
+fi;
+# We have Radiance and hdrgen available!
+
+# With all of the images selected, generate the HDR image.
+hdrgen ${INPUT_FILES[@]} -o $TEMP_FILENAME -r $CONFIG_CAMRESP_PATH -a -e -f -g
+
+echo "Compiled the initial HDR image, now processing..."
+
+# Exposure Nullification
+echo "Nullifying Exposures..."
+
+ra_xyze -r -o $TEMP_FILENAME > $TEMP_FILENAME_ALT
+
+# Image Cropping
+echo "Cropping Image..."
+
+pcompos -x $CONFIG_FISHEYE_DIAM -y $CONFIG_FISHEYE_DIAM $TEMP_FILENAME_ALT -$CONFIG_CROP_X -$CONFIG_CROP_Y > $TEMP_FILENAME
+
+# Vignetting Correction
+echo "Correcting the vignetting..."
+
+pcomb -f $CONFIG_VIG_PATH -e "diameter=$CONFIG_FISHEYE_DIAM" $TEMP_FILENAME > $TEMP_FILENAME_ALT
+
+# Resize Image
+echo "Rescaling Image..."
+
+pfilt -1 -x $CONFIG_OUT_X -y $CONFIG_OUT_Y $TEMP_FILENAME_ALT > $TEMP_FILENAME
+
+# Fisheye Correction
+echo "Correcting Fisheye..."
+
+pcomb -f $CONFIG_FISHEYE_PATH $TEMP_FILENAME > $TEMP_FILENAME_ALT
+
+# Applying Neutral Density Filter
+echo "Applying Neutral Density Filter..."
+
+pcomb -f $CONFIG_ND_PATH $TEMP_FILENAME_ALT > $TEMP_FILENAME
+
+# Photometic Adjustments
+echo "Applying Photometic Adjustments..."
+
+pcomb -h -f $CONFIG_CALFAC_PATH $TEMP_FILENAME > $TEMP_FILENAME_ALT
+
+# Edit the header
+echo "Editing the header..."
+
+getinfo < $TEMP_FILENAME_ALT | sed "/VIEW/d" > /dev/null
+
+# Adjust for real viewing angle
+echo "Adjusting for real viewing angle..."
+
+getinfo -a "VIEW = -vta -view_angle_verticle $CONFIG_VA_V -view_angle_horizontal $CONFIG_VA_H" < $TEMP_FILENAME_ALT > $TEMP_FILENAME
```

Comment:
This one might differ: `-vt?` instead of `-vta`

Reply:
Correct command is indeed: `getinfo -a "VIEW= -vta -vv $CONFIG_VA_V -vh $CONFIG_VA_H" < $TEMP_FILENAME_ALT > $TEMP_FILENAME
`